tooling/miniutil/src/build/global.rs

```diff
@@ -1,8 +1,8 @@
 use crate::build::*;
 
 impl ProgramBuilder {
-    pub fn declare_global_zero_initialized<T: TypeConv>(&mut self) -> PlaceExpr {
-        let bytes = List::from_elem(Some(0), T::get_size().bytes());
+    pub fn declare_global_zero_initialized<T: TypeConv + Sized>(&mut self) -> PlaceExpr {
```

Comment:
The `+ Sized` is redundant, generic types are by default required to be sized.

Reply:
Yeah sure, I actually knew that, but somehow got confused anyway, thinking `?Sized` would be a supertrait of `TypeConv`, which a) isn't necessary, since the implicit Sized by default of course doesn't apply to trait and b) isn't actually legal rust. Reverting of course.